### ISSUE TITLE
document new "deploy notebook" feature

### ISSUE THREAD

**Issue by edublancas** [ploomber/doc#52]
see: https://github.com/ploomber/jupysql-plugin/issues/93

important: once the documentation is ready, I need to open another PR to the frontend repository so the "new" button takes users to the documentation, see: https://github.com/ploomber/cloud-frontend/pull/246#discussion_r1420156202

**Comment by edublancas** [comment]
this has been fixed
